app/Http/Controllers/SubmissionController.php

```diff
@@ -62,7 +63,9 @@ private function failProcessing(?string $filename, int $outResponseCode, string
         if ($filename !== null) {
             Storage::move('inbox/' . $filename, 'failed/' . $filename);
         }
-        abort($outResponseCode, $outMessage);
+        if (config('cdash.validate_submissions') === SubmissionValidationType::REJECT) {
+            abort($outResponseCode, $outMessage);
+        }
```

Comment:
Won't this affect non-validation failures as well?  We always want to know if a server error happens I think.

Reply:
Almost certainly.  I think this was more of a leftover from a previous workflow.  Thanks !